src/mail/view/MailView.ts

```diff
@@ -556,6 +565,9 @@ export class MailView extends BaseTopLevelView implements TopLevelView<MailViewA
 	}
 
 	private setUrl(url: string) {
+		if (typeof url === "undefined") {
+			throw new Error("undefined mail url!")
```

Comment:
is this a shouldn't-happen programming error?

Reply:
I swapped it for `assertNotNull` which cannot use `ProgrammingError` unfortunately but maybe it even should have it's own `AssertionError` or NPE of sorts.

Reply:
even better!